distribution/openrct2.d.ts

```diff
@@ -3324,6 +3324,20 @@ declare global {
          * The total number of frames in the current animation.
          */
         readonly animationLength: number;
+
+        lawnsMown: number | null;
+
+        gardensWatered: number | null;
+
+        litterSwept: number | null;
+
+        binsEmptied: number | null;
+
+        ridesFixed: number | null;
+
+        ridesInspected: number | null;
+
+        vandalsStopped: number | null;
```

Comment:
Another thought, why are these properties writable? I don't really see a benefit / use case, but I also don't think that it will cause any issues.

Reply:
This has not been adressed yet, can we have a (short) discussion on it?

Reply:
I’m sorry, I overlooked your comment.

TBH I can’t think of any use case for them to be writable. Should we make them read-only until someone brings it up?

Reply:
Yes, I think that is better.